src/env/env.rs

```diff
@@ -44,6 +44,10 @@ fn print_env(null: bool) {
     }
 }
 
+fn split_string(s: &str) -> Vec<String> {
+    s.split_whitespace().map(|x| x.to_owned()).collect::<Vec<String>>()
```

Comment:
Rather than allocating a `Vec`, you can remove the `.collect()` and make this return `impl Iterator<Item = String>`.